fhir-bench-orchestrator/src/servers/mod.rs

```diff
@@ -1,21 +1,27 @@
 //! TODO
 
 use anyhow::Result;
-use serde::Serialize;
+use serde::{Deserialize,Serialize};
 use url::Url;
 
 mod hapi_jpa;
 
 /// Represents the unique name of a FHIR server implementation.
-#[derive(Debug, PartialEq, Clone, Serialize)]
-pub struct ServerName(pub &'static str);
+#[derive(Debug, PartialEq, Clone, Deserialize, Serialize)]
+pub struct ServerName(pub String);
 
 impl std::fmt::Display for ServerName {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         write!(f, "{}", self.0)
     }
 }
 
+impl From<&str> for ServerName {
```

Comment:
Should have a comment here or on the struct demonstrating how to create `ServerName`s.